### PR TITLE
feat: add additional up sells in the plugin for increased awareness of pro version

### LINKED ISSUE #4400: feat: add additional up sells in the plugin for increased awareness of pro version
# Feature Request

In order to increase new customer counts we should add new up sells into the plugin that are not overly intrusive but still beneficial to bringing customers onto our site.

### 1. Update "Add-ons" link so it's more prominent

Ensure that the "Add-ons" link is more visible and opens in a new tab so that we can then send to a landing page on our site specifically for users coming from the plugin.

The current page that opens now can't be measured. We can use UTM links within the new one and a tiny URL to track the clicks.

**Current UI**

**Proposed UI**

1. Displays Green when *no add-ons are activated*.
2. When an add-on is activated it goes back to the normal color as to not distract further from the UI.

### 2. Upsell Recurring Donations 

If we detect that the user is not using Recurring then we should upsell them with a nice subtle banner.

1. Within the Donation Form Edit Screen 

Recurring Upsell

2. ....

### 3. Dashboard Widget 

If no add-ons are activated the Dashboard widget can show an upsell to view our catalog. 

[Concept coming...]

---

Related issue: https://github.com/impress-org/givewp/issues/3660

### PR DESCRIPTION
**Pull request by ravinderk**
## Description
This PR will resolve #4400.

Note: This will only fix points 1 and 2 mentioned in the issue description.

## How Has This Been Tested?
https://www.loom.com/share/1de57b24d4944203851053675e175f89

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

**Comment by DevinWalker** [comment]
@ravinderk can you please ensure that this notice only appears on the first tab?

File: includes/admin/admin-actions.php [hunk 1]
```diff
@@ -1509,6 +1509,44 @@ function give_admin_quick_js() {
 
 add_action( 'admin_head', 'give_admin_quick_js' );
 
+/**
+ * Admin Footer JS
+ *
+ * @since 2.6.0
+ */
+function give_admin_footer_quick_js() {
```
Comment (DevinWalker) [inline includes/admin/admin-actions.php]: Please update the name of this function so it is more descriptive to add upsell JS.

File: includes/admin/admin-pages.php [hunk 2]
```diff
@@ -102,21 +102,16 @@ function give_add_options_links() {
...
-	$give_add_ons_page = add_submenu_page(
-		'edit.php?post_type=give_forms',
-		esc_html__( 'GiveWP Add-ons', 'give' ),
+	// Add-ons
+	$submenu['edit.php?post_type=give_forms'][] = array(
 		esc_html__( 'Add-ons', 'give' ),
 		'install_plugins',
-		'give-addons',
-		'give_add_ons_page'
+		esc_url( 'https://givewp.com/addons/' ),
```
Comment (DevinWalker) [inline includes/admin/admin-pages.php]: Add a filter here so folks like Astra can easily modify to add an affiliate link.

Comment (DevinWalker) [reply includes/admin/admin-pages.php]: Name of the filter: `give_addon_menu_item_url`.

**Comment by ravinderk** [comment]
@DevinWalker Changes implemented as per your suggestion.

**Pull request merged**